src/main/java/frc/team2412/robot/subsystems/IntakeSubsystem.java

```diff
@@ -218,8 +229,13 @@ public boolean indexSensorHasNote() {
 		return indexSensor.isPressed();
 	}
 
+	public boolean debouncedFeederSensor() {
+		return feederSensorDebouncer.calculate(feederSensor.isPressed());
```

Comment:
To simplify the rest of the code & de-bounce both feeder sensors, try this:
```suggestion
		boolean feederSensorSignal = feederSensorDebouncer.calculate(feederSensor.isPressed());
		boolean feederSensorIRSignal = feederSensorIRDebouncer.calculate(feederSensorIR.get());
		return feederSensorSignal  || feederSensorIRSignal ;
```

Reply:
And remove the use of `feederSensorIR` from `feederSensorHasNote()`